staging/src/github.com/openshift/oc/pkg/cli/admin/release/mirror.go

```diff
@@ -442,6 +452,14 @@ func (o *MirrorOptions) Run() error {
 	} else {
 		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirrored to: %s\n", to, o.To)
 	}
+	
+	// Print list of repos mirrored from for use with installing or upgrading when disconnected
```

Comment:
the goal for the output is to be directly useful for installation and upgrade.

```
To use the new mirrored release-image to install, provide the following list of release-image sources to the installer using the install-config.yaml:
imageContentSource:
- sources:
  - <mirrored repo>
  - <origin release-image repo>
  - <repos referenced in the release-image>
  ...
```

Let's make sure the order is same as the example above..

Reply:
Updated output as requested.  PR description is updated with example.